packages/react-components/src/components/styles/HorizontalGallery.styles.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getTheme, IStyle } from '@fluentui/react';
+
+const theme = getTheme();
```

Comment:
Question: if someone updates the theme will it be updated? (i.e. does getTheme need called inside the render to prevent it being a stale value)

Reply:
I think it will. In storybook when we change theme, this component updates correctly.
@mgamis-msft could you verify?

Reply:
No, `getTheme` will not get updates to theme updates. `getTheme` is essentially a constant value.